nitro-shim/tools/viproxy/go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/brave-experiments/ia2-parent/viproxy
+
+go 1.17
```

Comment:
bump 1.18

Reply:
Should be addressed in https://github.com/brave-intl/bat-go/commit/4e928e5b0d36c6f5e6b195ea73e79b07a86690a1.